docs/authentication.md

```diff
@@ -184,7 +186,7 @@ magic.auth.mac(message)
   });
 });
 
-// supplied key
+// supplied key; 48 bytes Buffer or hex encoded string
```

Comment:
```suggestion
// Authenticating a message with a supplied key,
// which must be a 48-byte Buffer or hex-encoded string.
```

Reply:
And for [line 163](https://github.com/auth0/magic/pull/60/files#diff-361f92eed2eaa3203409b5ccbdfa0802R163), from `// key generation` to:
```
// Authenticating a message with a newly generated key
```

Reply:
[Line 220](https://github.com/auth0/magic/pull/60/files#diff-361f92eed2eaa3203409b5ccbdfa0802R220) also just says `// supplied key`. Hope you don't mind I'm looking outside of current changes. =)